.github/workflows/workflow.yml

```diff
@@ -19,12 +19,48 @@ jobs:
         with:
           path: poetry.lock
           key: ${{ github.sha }}-${{ matrix.python-version }}
-  pytest:
+  pytest-dynalite:
     needs: lock
     strategy:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
     runs-on: "ubuntu-latest"
+    services:
+      dynalite:
+        image: dimaqq/dynalite:latest
+        ports:
+          - 8000:8000
+    env:
+      DYNAMODB_URL: http://localhost:8000
+      AWS_ACCESS_KEY_ID: dummy
+      AWS_SECRET_ACCESS_KEY: dummy
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: actions/cache@v2
+        with:
+          path: poetry.lock
+          key: ${{ github.sha }}-${{ matrix.python-version }}
+      - run: curl -sSL https://install.python-poetry.org | python3 -
+      - run: poetry install --extras aiohttp --extras httpx
+      - run: poetry run pytest --verbose
+  pytest-dynamodb-local:
+    needs: lock
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+    runs-on: "ubuntu-latest"
+    services:
+      dynamodb:
+        image: amazon/dynamodb-local:latest
+        ports:
+          - 8000:8000
```

Comment:
Ugh that's a bummer that services don't grok matrix....
I guess there's another alternative:
* 1 job
* has 2 services on 2 ports
* matrix which port is used by `pytest`

Perhaps that's too clever though -- hard to tell what fails if it's `port 8001` 🤔